site/content/en/docs/Getting Started/create-gameserver.md

```diff
@@ -31,7 +31,10 @@ For the purpose of this guide we're going to use the {{< ghlink href="examples/s
 Let's create a GameServer using the following command :
 
 ```
+# Linux Game Server
```

Comment:
Please surround this with 

{{% feature publishVersion="1.14.0" %}}
...
{{% /feature %}}

so that it goes live with the next release (when we actually have an agones sdkserver that will run on windows nodes). 

For ``` sections you need to copy / paste the whole block because the conditional won't work inside of the escaped section.